### ISSUE TITLE
log and accessibility issues

### ISSUE THREAD

**Issue by afr-ux** [home-assistant/frontend#24693]
### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [x] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

In registry screen:
http://homeassistant:8123/logbook
There are some information not accessible with screenreaders:
1- There are 2 dates in the same read-only edit box. It is not possible open calendar with screenreader. It is neccessary 2 different elements and elements that I can use it, for example, links or buttons for my screenreader.
2- When calendar show, it is not modal screen. It is neccessary modal screen to separate information for my screenreader.
3- I can do intro in a day of this calendar. Nothing happen if I try select a day with keyboard with screenreader.
I test it in chrome, firefox, edge with JAWS and NVDA in Windows 10.
Thanks.

### Describe the behavior you expected

Read information succesfully by screenreader

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

All

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### Javascript errors shown in your browser console/inspector

```txt

```

### Additional information

In registry screen:
http://homeassistant:8123/logbook
There are some information not accessible with screenreaders:
1- There are 2 dates in the same read-only edit box. It is not possible open calendar with screenreader. It is neccessary 2 different elements and elements that I can use it, for example, links or buttons for my screenreader.
2- When calendar show, it is not modal screen. It is neccessary modal screen to separate information for my screenreader.
3- I can do intro in a day of this calendar. Nothing happen if I try select a day with keyboard with screenreader.
I test it in chrome, firefox, edge with JAWS and NVDA in Windows 10.
Thanks.

**Comment by karwosts** [comment]
Thank you for this report, but it is a duplicate of existing issue #14141, so I will close this as duplicate.
